config/default.yml

```diff
@@ -151,6 +151,7 @@ PreCommit:
     flags: ['-IHn', "\t"]
     exclude:
       - '**/*.go'
+      - 'Godeps/Godeps.json'
```

Comment:
We seem to have a bit of a mix in this file, and I don't think it matters much, but I would personally go with alphabetization.